### DISCUSSION TITLE
Ability to automatically hitsound objects for consistent sounds in the music on the editor

### DISCUSSION THREAD

**Discussion by gustarmartins** [ppy/osu#16615] [Ideas]
https://twitter.com/osu_MkGuh/status/1066681571074297857

I did this tweet in 2018, and peppy seemed to be a fan of the idea. I don't know if this is on the to-do list, but in my opinion it is a definitely needed thing to have natively on the editor.

I think this has been explained well, so I probably don't need to get into more details, but I can if it's not clear enough yet.

**Comment by peppy** [comment]
Copying hitsounds across hitobjects will be something we eventually get to. sure.
